pkg/registry/core/namespace/storage/storage.go

```diff
@@ -294,6 +297,10 @@ func (r *REST) StorageVersion() runtime.GroupVersioner {
 	return r.store.StorageVersion()
 }
 
+// ResetFields implements rest.ResetFieldsProvider
+func (r *REST) ResetFields() rest.ResetFields {
```

Comment:
"Reset" is a verb and that makes this read like a command instead of a Getter. Can we think of a noun? 

"GetResetFields" is a slight improvement :/

How about "UnchangeableFields" for this concept?